packages/inscription-view/src/components/widgets/inscription-tabs/InscriptionTabs.css

```
@@ -0,0 +1,93 @@
+.part-tabs-root {
+  display: flex;
+  flex-direction: column;
+  padding: var(--size-2);
+}
+
+.part-tabs-list {
+  display: flex;
+  overflow: hidden;
+  padding: var(--size-1);
+  background-color: var(--N25);
+  border: var(--basic-border);
+  border-radius: var(--border-r3);
+  margin-bottom: var(--size-3);
+  container: tabs-list / inline-size;
+}
+
+.part-tabs-trigger {
+  all: unset;
+  font-family: inherit;
+  height: var(--size-2);
+  padding: var(--size-3) var(--size-1) var(--size-3) var(--size-1);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  white-space: nowrap;
+  font-size: 12px;
+  font-weight: bold;
+  line-height: 1;
+  color: var(--N700);
+  user-select: none;
+  cursor: pointer;
+  flex: 1;
+  gap: var(--size-1);
+  min-width: 2rem;
+  container: tab-trigger / inline-size;
+}
+
+.part-tabs-list:has(.part-tabs-trigger:focus-visible) {
+  box-shadow: var(--focus-shadow);
+}
+
+.part-tabs-trigger:hover {
+  color: var(--P300);
+}
+
+.part-tabs-trigger[data-state='active'] {
+  border: 1px solid var(--P75);
+  border-radius: var(--border-r2);
+  background-color: var(--P50);
+  color: var(--P300);
+  min-width: 6.5rem;
+}
+
+.part-tabs-content {
+  display: flex;
+  flex-direction: column;
+  gap: var(--size-3);
+}
+
+.part-tabs-trigger i {
+  font-size: 16px;
+}
+
+@container tab-trigger (width <= 6.5rem) {
+  .tab-label {
+    display: none;
+  }
+
+  .part-tabs-trigger[data-state='active'] .tab-label {
+    display: inline;
+  }
```

I would combine this two
```suggestion
  .part-tabs-trigger:not([data-state='active']) .tab-label {
    display: none;
  }
```